src/components/Dropdown/dropdown.css

```diff
@@ -2,8 +2,55 @@
 @import "src/css/variables/fonts.css";
 @import "src/css/variables/colors.css";
 
-.y-dropdown .ms-Dropdown-label{
+.y-dropdown .ms-Dropdown-label {
   @extend text__mediumSub;
   font-weight: $fontWeight__bold;
   color: $textColor__secondary;
 }
+
+.y-dropdown .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown-caretDownWrapper {
+  height: 36px;
+  line-height: 3.3rem;
+}
+
+.y-dropdown .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .y-dropdown__caretDown {
+  color: $color__river;
+}
+
+.y-dropdown .ms-Dropdown:hover .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:focus .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title {
+  border-color: $color__lake;
+}
+
+.y-dropdown .ms-Dropdown:focus .ms-Dropdown-title {
+  border-color: $color__lake;
+}
+
+/* NOTE: y-dropdown cannot be used here since the options are not a descendent
+ * of dropdown element
+ */
+.ms-Dropdown-items .ms-Dropdown-item:hover,
```

Comment:
@bryan-gislason maybe we need to also provide an `onRenderContainer` method, and put our `y-dropdown` class on the wrapper instead of passing it in as a className prop?

Reply:
oh duh ignore this comment, we're already wrapping Dropdown in a div with our classname. so it looks like the items are rendered in a separate Layer? as @annr suggested maybe passing in a className to Dropdown would pass it along, or maybe there's another way to pass a className into the Layer wrapper?

Reply:
ok i looked at Drodpown and figured this out one. you can pass a `calloutProps` prop to Fabric's Dropdown like this:

```
<Dropdown
  ...
  calloutProps={{ className: 'y-dropdown--callout }}
```

then you could use the className you pass in your selectors